lighthouse-cli/test/fixtures/source-map/source-map-tester.html

```diff
@@ -0,0 +1,31 @@
+<!--
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+-->
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Source maps tester</title>
+</head>
+
+<body>
+    <script>
+        console.log('a');
+        // Test that source maps work.
+        //# sourceMappingURL=http://localhost:10200/source-map/script.js.map
```

Comment:
wait a sec I could've sworn this thing had smoketests did they get removed at some point in review of #9101  ?

Reply:
The unit tests covered the same stuff but better so I removed it.